features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -525,6 +525,13 @@
 				<AttributeID>failedLoginAttempts</AttributeID>
 				<Description>Failed Login Attempts</Description>
 			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/identity/failedLoginAttemptsBeforeSuccess</ClaimURI>
+				<DisplayName>Failed Attempts Before Success</DisplayName>
```

Comment:
```suggestion
				<DisplayName>Failed Attempts Before Last Login</DisplayName>
```
Or 
> Failed attempts between last two logins

Reply:
Display name should be less than 30 characters.